packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesController.kt

```diff
@@ -281,12 +286,14 @@ class UpdatesController private constructor(
                 throw AssertionError("Background update with error status must have a nonnull update object")
               }
               remoteLoadStatus = ErrorRecoveryDelegate.RemoteLoadStatus.NEW_UPDATE_LOADED
+              logger.info("UpdatesController onBackgroundUpdateFinished: Update available", UpdatesErrorCode.None)
```

Comment:
should the info method not have an error code argument?

Reply:
That is not an error

Reply:
Right, what I mean is should `logger.info` only take one argument since all callsites will just be passing `UpdatesErrorCode.None` as the second arg?

Reply:
I think that `UpdatesErrorCode.None` is the default value already for iOS, so we should do the same for the Kotlin code in Android.